src/timeseries.jl

```diff
@@ -0,0 +1,184 @@
+export veltimeseries
+
+"""
+    veltimeseries(notes, grid = 0:1//notes.tpq:1; zeros = false, av = false)
+
+Generate a time series of velocities out of given `notes`, by
+also quantizing on given grid (see [`quantize`](@ref)).
+### Keyword Arguments
+- `zeros` : If `true`, put zeros on gridpoints where no note occurs. If `false` then
+  simply stich the individual entries.
+- `av`: If multiple notes appear at one gridpoint, the highest velocity is
+  chosen by default (`false`).
+  Set this to `true` to average over different velocities at that grid point.
+"""
+function veltimeseries(notes::MIDI.Notes, grid = 0:1//notes.tpq:1; zeros::Bool = false, av::Bool = false)
+
+    #quantize if needed
+    if grid != 0:1//notes.tpq:1
+        quantize!(notes, grid)
+
+        #select if to put zeros and do timeseries
+        if zeros
```

Comment:
Why do 4 cases exist here? Why do you separate the notes on whether you "quantized"? There is no need. Make it shorter and only have 2 cases. 

Also, we will always quantize anyway; same story for the "default" grid.